<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Modo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tet_plus_mode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Umbral</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFT</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pausa">
        <property name="text">
         <string>Pausa</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>30000</number>
        </property>
        <property name="autoRepeatInterval">
         <number>10000</number>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="autoRange">
        <property name="text">
         <string>autoRange</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tetrodo:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="tet_plus_selec">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Canal:  </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="canal_selec">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Umbral:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="c_auto_umbral">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="c_manual_umbral">
        <property name="text">
         <string>Manual</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="GraphicsView" name="plus_grid_fr">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>10000</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="GraphicsView" name="plus_grid">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10000</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="GraphicsView" name="espacio_pg"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuGrabar">
    <property name="title">
     <string>Grabar</string>
    </property>
    <addaction name="actionComenzar"/>
    <addaction name="actionDetener"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuGrabar"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDetener">
   <property name="text">
    <string>Detener</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionComenzar">
   <property name="text">
    <string>Comenzar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionDetener</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionDetener()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionSalir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNuevo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionNuevo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionComenzar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionComenzar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_actionDetener()</slot>
  <slot>on_actionSalir()</slot>
  <slot>on_actionNuevo()</slot>
  <slot>on_actionComenzar()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
